### ISSUE TITLE
static dividing scheduler very slow

### ISSUE THREAD

**Issue by DanielXLee** [clusternet/clusternet#335]
create a nginx deployment and subscription with a static dividing scheduler,  the scheduler needs to wait about 17 minutes to complete the schedule

#### What happened:

```console
I0517 11:57:24.342400       1 scheduler.go:431] no updates on the spec of Subscription clusternet-subs/dividing-demo, skipping syncing
I0517 11:57:24.355973       1 scheduler.go:431] no updates on the spec of Subscription clusternet-subs/dividing-demo, skipping syncing
I0517 11:57:29.995871       1 scheduler.go:497] no updates on the labels/taints of ManagedCluster clusternet-cc8c2/cls-6nyhdo0q, skipping syncing
I0517 11:57:55.333693       1 scheduler.go:497] no updates on the labels/taints of ManagedCluster clusternet-s7cth/cls-bhblikxk, skipping syncing
I0517 11:57:56.745481       1 scheduler.go:497] no updates on the labels/taints of ManagedCluster clusternet-8p5dj/cls-chlxiz3o, skipping syncing
I0517 11:58:23.501070       1 scheduler.go:497] no updates on the labels/taints of ManagedCluster clusternet-lqchp/cls-2dzczntm, skipping syncing
....
I0517 12:13:58.564074       1 scheduler.go:497] no updates on the labels/taints of ManagedCluster clusternet-f2skt/cls-qdfkwvkk, skipping syncing
I0517 12:14:04.330711       1 scheduler.go:239] "Attempting to schedule subscription" subscription="clusternet-subs/dividing-demo"
I0517 12:14:04.330902       1 static.go:50] "Attempting to assign replicas to clusters" subscription="clusternet-subs/dividing-demo" clusters=[clusternet-s7cth/cls-bhblikxk clusternet-cc8c2/cls-6nyhdo0q]
I0517 12:14:04.331025       1 default_binder.go:52] "Attempting to bind subscription to clusters" subscription="clusternet-subs/dividing-demo" clusters=[clusternet-s7cth/cls-bhblikxk clusternet-cc8c2/cls-6nyhdo0q]
I0517 12:14:04.343684       1 scheduler.go:431] no updates on the spec of Subscription clusternet-subs/dividing-demo, skipping syncing
```

#### Environment:
- Clusternet version:
    - Clusternet-agent version (user `clusternet-agent --version=json`):
    - Clusternet-hub version (user `clusternet-hub --version=json`):
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Others:

**Comment by silenceper** [comment]
I also encountered this problem.
scheduler queue blocking?

**Comment by dixudx** [comment]
Have you run scheduler with leader elections?

**Comment by silenceper** [comment]
The default settings are used, and the leader election should have been turned on.

I'll check first if it's a problem

**Comment by dixudx** [comment]
This has been fixed in #399.
